Library/Formula/lightning.rb

```diff
@@ -13,8 +11,41 @@ class Lightning < Formula
     sha1 "eb8db8f32ceddaddaf445bb45268d13d753f89ba" => :lion
   end
 
+  option "with-check", "run the test suite when building"
+
   def install
-    system "./configure","--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "check" if build.with? "check"
```

Comment:
I guess I am pretty skeptical about adding "with-check" options. Do people actually use them? What good is it if we never run it in CI? Given we have bottles, I don't seem the harm in running it unconditionally.